spec/support/js_console.rb

```diff
@@ -0,0 +1,22 @@
+module JsConsole
+  def console_messages
+    page.driver.phantomjs_logger.rewind
+    page.driver.phantomjs_logger.read.split("\n")
+  end
+
+  def clear_console_messages
+    page.driver.phantomjs_logger.rewind if defined?(page)
```

Comment:
How does this exactly work? If this really clears console messages, won't `#console_messages` always return an empty array?